services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/flows/FlowCrudSpec.groovy

```diff
@@ -437,6 +437,27 @@ class FlowCrudSpec extends BaseSpecification {
         }
     }
 
+    def "Able to validate flow with protected path"() {
```

Comment:
If this is an all-positive happy path for protected path, I think it worth adding switch validation for each involved switch (in both main and protected paths) as well as traffic examination part. Also try not to overlap with existing tests, I kind of lost count of what is already covered -_-

Reply:
new step(` when: "Delete rule of protected path on the srcSwitch"`) was added

Reply:
Ok the test is now completely reworked. This is not about FlowCrud anymore. It used to be a positive scenario for creating a flow with protected path but now it is a whole different test that removes rules from the flow. Please see other tests in this spec to comply with the context here. Current implementation is closer to flow validation tests (not flow crud)

Reply:
sorry for confusing
two separate tests were created
and new validation for each involved switches(main and protected) was added